boa3_test/tests/test_interop.py

```diff
@@ -548,76 +548,174 @@ def test_invocation_counter_cant_assign(self):
         self.assertEqual(expected_output, output)
 
     def test_ripemd160_str(self):
-        string = String('test').to_bytes()
         expected_output = (
-            Opcode.PUSHDATA1
-            + Integer(len(string)).to_byte_array(min_length=1)
-            + string
+            Opcode.INITSLOT
+            + b'\x00\x01'
+            + Opcode.LDARG0
             + Opcode.SYSCALL
             + Interop.Ripemd160.interop_method_hash
-            + Opcode.DROP
-            + Opcode.PUSHNULL
             + Opcode.RET
         )
 
         path = '%s/boa3_test/test_sc/interop_test/Ripemd160Str.py' % self.dirname
         output = Boa3.compile(path)
         self.assertEqual(expected_output, output)
 
-    def test_hash160_str(self):
-        string = String('test').to_bytes()
-        expected_output = (
-            Opcode.PUSHDATA1
-            + Integer(len(string)).to_byte_array(min_length=1)
-            + string
-            + Opcode.SYSCALL
-            + Interop.Sha256.interop_method_hash
-            + Opcode.SYSCALL
-            + Interop.Ripemd160.interop_method_hash
-            + Opcode.DROP
-            + Opcode.PUSHNULL
-            + Opcode.RET
-        )
+        import hashlib
+        engine = TestEngine(self.dirname)
+        expected_result = hashlib.new('ripemd160')
+        expected_result.update(b'unit test')
+        result = self.run_smart_contract(engine, path, 'Main', 'unit test')
+        self.assertEqual(expected_result.digest(), result)
+
+        expected_result = hashlib.new('ripemd160')
+        expected_result.update(b'')
+        result = self.run_smart_contract(engine, path, 'Main', '')
+        self.assertEqual(expected_result.digest(), result)
 
+    def test_ripemd160_int(self):
+        import hashlib
+        path = '%s/boa3_test/test_sc/interop_test/Ripemd160Int.py' % self.dirname
+        engine = TestEngine(self.dirname)
+        expected_result = hashlib.new('ripemd160')
+        expected_result.update(bytes([10]))
```

Comment:
Also, instead of creating bytes in this way, you can use ``Integer(10).to_byte_array()``

This works the same way as String, it's meant to be used to keep a standardized conversion in the whole code between ``int`` and ``bytes`` types